data-collection/data-collection/View Controllers/Rich Popup View Controllers/Rich Popup View Controller/RichPopupViewController.swift

```diff
@@ -179,7 +179,7 @@ class RichPopupViewController: SegmentedViewController {
         
         if self.popupManager.isEditing {
             // Add Cancel button. Will hide back bar button, if there is one.
-            navigationItem.leftBarButtonItem = UIBarButtonItem(title: "Cancel", style: .plain, target: self, action: #selector(userRequestsCancelEditingPopup(_:)))
+            navigationItem.leftBarButtonItem = UIBarButtonItem(barButtonSystemItem: .cancel, target: self, action: #selector(userRequestsCancelEditingPopup(_:)))
```

Comment:
Making this suggestion because it concerns the same single line of code changed by this PR.

```suggestion
            navigationItem.leftBarButtonItem = UIBarButtonItem(barButtonSystemItem: .cancel, target: self, action: #selector(userRequestsCancelEditingPopup))
```

Reply:
Funny. Both autocomplete and Apple's explanation in their [documentation](https://developer.apple.com/documentation/swift/using_objective-c_runtime_features_in_swift) include the parentheses, but then the example code they give doesn't. It compiles and works the same either way, so I took your suggestion.